src/tls/README.md

```diff
@@ -87,14 +87,21 @@ see the [guide](../../doc/guide/cert-authentication.xml) and
 [manpage](../../doc/man/cockpit.conf.xml). Then cockpit-tls will ask the
 browser for a client certificate.
 
-If cockpit-ws sees that cockpit-tls exports a certificate for its connection
-(by checking its cgroup instance name, which is the certificate fingerprint,
-and checking /run/cockpit/tls for it), then it will request the `tls-cert`
-authentication schema from cockpit-session, instead of the usual `basic` or
-`gssapi`. cockpit-session then does *not* set a PAM user name, and
```

Comment:
I think this is still relevant -- you may still want to control/deviate authentication schemes in cockpit.conf.